src/main/kotlin/com/terraformation/backend/eventlog/FieldsUpdatedEvent.kt

```diff
@@ -0,0 +1,44 @@
+package com.terraformation.backend.eventlog
+
+/**
+ * Common interface for events that represent one or more fields of an entity being updated. This is
+ * used to render [PersistentEvent]s into API payloads for event log queries. Update events should
+ * either implement this interface or be handled explicitly in
+ * [EventLogPayloadTransformer.getActions].
+ */
+interface FieldsUpdatedEvent {
+  /**
+   * Returns a list of all the fields that were updated by the operation this event represents.
+   * Fields that weren't modified shouldn't be included in the list. If the event represents a
+   * single field change, this should return a list of length 1.
+   */
+  fun listUpdatedFields(): List<UpdatedField>
```

Comment:
Does this need to be returned in a specific chronological order? Or is this more a representation of a single event that updated multiple fields? I'm assuming the latter since the interface is `FieldsUpdatedEvent` instead of plural; it's probably just my lack of being in this head space causing me to be confused.